src/blocks/content-gate-countdown/class-content-gate-countdown-block.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * Content Gate Countdown Block
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+use Newspack\Memberships;
+use Newspack\Memberships\Metering;
+
+/**
+ * Content Gate Countdown Block class.
+ */
+class Content_Gate_Countdown_Block {
+	/**
+	 * Initialize the block.
+	 */
+	public static function init() {
+		add_action( 'init', [ __CLASS__, 'register_block' ] );
+		add_action( 'wp_enqueue_scripts', [ __CLASS__, 'enqueue_scripts' ] );
+	}
+
+	/**
+	 * Enqueue block scripts and styles.
+	 *
+	 * @return void
+	 */
+	public static function enqueue_scripts() {
+		if ( ! Memberships::is_active() || ! is_singular() ) {
+			return;
+		}
+		wp_enqueue_style(
+			'newspack-content-gate-countdown-block',
+			\Newspack\Newspack::plugin_url() . '/dist/content-gate-countdown-block.css',
```

Comment:
Not sure why, but I'm getting 404 for this asset.

Reply:
Looks like the styles for this are being bundled into commons by webpack. Not sure if this is what you are seeing. On my end, there is no `content-gate-countdown-block.css` being fetched. Instead the styles live in `commons.css`

Reply:
It's getting added to the markup and throwing 404:

<img width="555" height="99" alt="image" src="https://github.com/user-attachments/assets/7804392e-dc88-4bc3-ae75-af12f93e940c" />